how-to-use-azureml/deployment/accelerated-models/README.md

```diff
@@ -0,0 +1,74 @@
+
+# Notebooks for Microsoft Azure Machine Learning Hardware Accelerated Models SDK
+
+Easily create and train a model using various deep neural networks (DNNs) as a featurizer for deployment to Azure or a Data Box Edge device for ultra-low latency inferencing using FPGA's. These models are currently available:
+
+* ResNet 50
+* ResNet 152
+* DenseNet-121
+* VGG-16
+* SSD-VGG  
+
+To learn more about the azureml-accel-model classes, see the section [Model Classes](#model-classes) below or the [Azure ML Python SDK documentation](https://docs.microsoft.com/en-us/python/api/overview/azure/ml/intro?view=azure-ml-py).
+
+### Step 1: Create an Azure ML workspace
+Follow [these instructions](https://docs.microsoft.com/en-us/azure/machine-learning/service/quickstart-create-workspace-with-python) to install the Azure ML SDK on your local machine, create an Azure ML workspace, and set up your notebook environment, which is required for the next step.
+  
+### Step 2: Install the Azure ML Accelerated Models SDK
+Once you have set up your environment, install the Azure ML Accel Models SDK. This package requires tensorflow >= 1.6,<2.0 to be installed. 
+
+If you already have tensorflow >= 1.6,<2.0 installed in your development environment, you can install the SDK package using: 
+
+``
+pip install azureml-accel-models
+``
+
+If you do not have tensorflow >= 1.6,<2.0 and are using a CPU-only development environment, our SDK with tensorflow can be installed using:
+
+``
+pip install azureml-accel-models[cpu]
+``
+
+If your machine supports GPU (for example, on an [Azure DSVM](https://docs.microsoft.com/en-us/azure/machine-learning/data-science-virtual-machine/overview)), then you can leverage the tensorflow-gpu functionality using:
+
+``
+pip install azureml-accel-models[gpu]
+``
+
+### Step 3: Follow our notebooks
+
+The notebooks in this repo walk through the following scenarios: 
+* [Quickstart](accelerated-models-quickstart.ipynb), deploy and inference a ResNet50 model trained on ImageNet
+* [Object Detection](accelerated-models-object-detection.ipynb), deploy and inference an SSD-VGG model that can do object detection
+* [Training models](accelerated-models-training.ipynb), train one of our accelerated models on the Kaggle Cats and Dogs dataset to see how to improve accuracy on custom datasets
+
+<a name="model-classes"></a>
+## Model Classes
+As stated above, we support 5 Accelerated Models. Here's more information on their input and output tensors.
+
+**Available models and output tensors**
+
+The available models and the corresponding default classifier output tensors are below. This is the value that you would use during inferencing if you used the default classifier.
+* Resnet50, QuantizedResnet50 (output_tensors = "classifier_1/resnet_v1_50/predictions/Softmax:0")
```

Comment:
If these are snippets user is supposed to enter as code please use ```backticks```

Reply:
is updated, can see at new commit https://github.com/Azure/MachineLearningNotebooks/pull/343/files#diff-e8750b7b6ee961ae966efa5c495a175d